pom.xml

```diff
@@ -132,14 +143,14 @@
                 <goals>
                   <goal>sign</goal>
                 </goals>
+                <configuration>
```

Comment:
Nice catch! I missed this mistake, and I suspect this is the cause of the build error we were seeing - namely that the gpg executable config was not being applied due to the config being in the wrong place...